### ISSUE TITLE
Black screen with logo

### ISSUE THREAD

**Issue by nzrin** [react-native-mapbox-gl/maps#620]
**Describe the bug**
Shows black screen with Mapbox logo.

**To Reproduce**
Running example app.

**Expected behavior**
Showing maps.

**Screenshots**
---

**Versions (please complete the following information):**
 - Platfrom: Android
 - Device: Device Opportunity
 - OS: [ColorOS
 - SDK Version 28
 - React Native Version 0.60

**Additional context**

**Comment by kristfal** [comment]
Sounds like an issue with access token or lack of network connection. 

This works for a lot of people and I can’t reproduce, so it is very likely an issue on your end.

**Comment by nzrin** [comment]
No. I've already created "accesstoken" file and run ```nom link``` but it is not working.

**Comment by albertpb** [comment]
Works with 7.0.10.
7.2.0 I have black screen.

Using Android with
"react": "16.9.0"
"react-native": "0.61.5"

**Comment by ferdicus** [comment]
Please try to build the example app and try to reproduce it there.
Come back with your findings. 

Thanks

**Comment by dac09** [comment]
I spent hours on this, turns out it was missing an access token, even though I was setting it on launch.

In my case for example, I had misconfigured react-native-config for android only, so android wasn’t receiving the access token value, while iOS was.

An easy way to debug what’s going on is to run

```
adb logcat | grep Mbgl
```

You might notice something else in those logs. Hope this will help you, or someone else.
